data_override/data_override.F90

```diff
@@ -605,6 +633,84 @@ subroutine get_domainUG(gridname, UGdomain, comp_domain)
      call mpp_get_compute_domain(SGdomain,comp_domain(1),comp_domain(2),comp_domain(3),comp_domain(4))
 end subroutine get_domainUG
 !===============================================================================================
+! <FUNCTION NAME="new_nudge3D">
+!   <DESCRIPTION>
+! This function constructs a 3D data_nudge object.
+!   </DESCRIPTION>
+function new_nudge3D(newgridname, newvarname, xbounds, ybounds, zbounds, newobscoeffname, newmodcoeffname)
```

Comment:
The function name `new_nudge3d` is sort of relative.  This won't be so "new" in 10 years.  `nudge3d_construct` might be a better choice of naming as it is more descriptive and is not relative to the current time.

Reply:
function name changed (also added nudge2d_construct)